cli/reactjs_jsx_ppx_v4.ml

```diff
@@ -1064,7 +1064,8 @@ let transformComponentDefinition nestedModules mapper structure returnStructures
           @
           if hasForwardRef then
             [
-              (Location.mknoloc (Lident "ref"), Pat.var (Location.mknoloc "ref"));
+              ( Location.mknoloc (Lident "ref"),
+                Pat.var ~attrs:optionalAttr (Location.mknoloc "ref") );
```

Comment:
Before continuing adding `mknoloc` that later need to be changed anyway.
The loc should be corresponding to something in the original source. The only question is whether the `loc_ghost` field is set to true or false.
(It would be set to true exactly once for each location used in the code produced by the ppx).

Reply:
Good catch. I'm gonna polish the loc corresponding to original source code.